File: Silver V/BOJ_2751/BOJ_2751.cpp
#include <algorithm>
#include <iostream>
#include <vector>

using namespace std;

int n;
vector<int> v;

int main()
{
    cin >> n;
    v.resize(n);
    for (auto& num : v) {
        cin >> num;
    }
    sort(v.begin(), v.end());
    for (auto& num : v) {
        cout << num << '\n';
    }

    return 0;
}

File: Silver V/BOJ_2751/BOJ_2751.py
import sys

n = int(input())
# Python에서의 sys.stdin.readline()은 C++에서의 ios::sync_with_stdio(0); cin.tie(0)의 역할을 한다.
info = [int(sys.stdin.readline()) for i in range(n)]
info.sort()

for i in info:
    print(i)
